cluster/src/main/java/org/apache/iotdb/cluster/query/LocalQueryExecutor.java

```diff
@@ -236,6 +280,105 @@ public long querySingleSeries(SingleSeriesQueryRequest request)
     }
   }
 
+  /**
+   * Create an IBatchReader of a path, register it in the query manager to get a reader id for it
+   * and send the id back to the requester. If the reader does not have any data, an id of -1 will
+   * be returned.
+   *
+   * @param request
+   */
+  public long queryMultSeries(MultSeriesQueryRequest request)
+      throws CheckConsistencyException, QueryProcessException, StorageEngineException, IOException {
+    logger.debug(
+        "{}: {} is querying {}, queryId: {}",
+        name,
+        request.getRequester(),
+        request.getPath(),
+        request.getQueryId());
+    dataGroupMember.syncLeaderWithConsistencyCheck(false);
+
+    List<PartialPath> paths = Lists.newArrayList();
+    request
+        .getPath()
+        .forEach(
+            fullPath -> {
+              try {
+                paths.add(new PartialPath(fullPath));
+              } catch (IllegalPathException e) {
+                // ignore
```

Comment:
I suggest to output exception information through logger, don’t ignore

Reply:
done